docs_src/getting-started/tools/Ch-CommandLineInterface.md

```diff
@@ -1,104 +1,714 @@
+$ edgex-cli deviceservice name -n TestDeviceService -v
+```
 
-The available CLI capabilities, commands and option flags are too many to share here.  Use –help with the CLI to get more information.
 
-## Example Usage
+### `edgex-cli deviceprofile list` 
+List device profiles
 
-By default, the CLI results are printed out in tabular format, but by providing –v flag the result is printed in plain json format.  Here are some common CLI command examples, using the EdgeX device as the object, to show how to query, create, update and delete EdgeX objects.  The same CLI command principals generally apply for the other EdgeX objects (such as profile, services, events, readings, etc.) 
+```
+$ edgex-cli deviceprofile list
+```
 
-### List
+### `edgex-cli deviceprofile add` 
+Add a device profile
 
-You can easily get all the available EdgeX devices or get a device with a specific name.
+```
+$ edgex-cli deviceprofile add -n TestProfile -r "[{\"name\": \"SwitchButton\",\"description\": \"Switch On/Off.\",\"properties\": {\"valueType\": \"String\",\"readWrite\": \"RW\",\"defaultValue\": \"On\",\"units\": \"On/Off\" } }]"  -c "[{\"name\": \"Switch\",\"readWrite\": \"RW\",\"resourceOperations\": [{\"deviceResource\": \"SwitchButton\",\"DefaultValue\": \"false\" }]} ]"
+{{{v2} 65d083cc-b876-4744-af65-59a00c63fc25  201} 4c0af6b0-4e83-4f3c-a574-dcea5f42d3f0}
 
-``` bash
-edgex-cli device list 
 ```
 
-![image](device-list.png)
+### `edgex-cli deviceprofile name` 
+Show information about a specifed device profile
+
+```
+$ edgex-cli deviceprofile name -n TestProfile
+Name         Description  Manufacturer  Model  Name
+TestProfile                                    TestProfile
 
-``` bash
-edgex-cli device list – name Random-Integer-Device
 ```
 
-![image](device-list-by-name.png)
+### `edgex-cli deviceprofile rm` 
+Remove a device profile
 
-``` bash
-edgex-cli device list – name Random-Integer-Device -v
+```
+$ edgex-cli deviceprofile rm -n TestProfile
 ```
 
-![image](device-list-json.png)
+### `edgex-cli device list` 
+List devices
 
-### Create
-Here is an example of how the CLI's interactive mode can be used together with inline parameters to populate the main structure of a new create device request. Note how all the inline provided parameters are pre-populated and the user is left to populate the rest of the needed values. 
+```
+$ edgex-cli device list
+```
+
+### `edgex-cli device add` 
+Add a new device
 
-``` bash
-edgex-cli device add --description CLI-Device --profileName Random-Integer-Device --operatingStatus ENABLED -i      (--interactive)
 ```
+$ edgex-cli device add -n TestDevice -p TestProfile -s TestDeviceService --protocols "{\"modbus-tcp\":{\"Address\": \"localhost\",\"Port\": \"1234\" }}"
+{{{v2} e912aa16-af4a-491d-993b-b0aeb8cd9c67  201} ae0e8b95-52fc-4778-892d-ae7e1127ed39}
 
-![image](create-device.png)
+```
 
-### Update
+### `edgex-cli device name` 
+Show information about a specified named device 
```

Comment:
is this an update or a list?

Reply:
Neither - the reference to removing "### Update" is not really relevant. The `edgex-cli device name` command just prints out information about a specific named device.